matRad_rayTracing.m

```diff
@@ -128,5 +129,7 @@
 % only take voxel inside the patient
 for i = 1:ct.numOfCtScen
     radDepthV{i} = radDepthCube{i}(V);
+    radDepthsMat{i} = radDepthCube{1};
```

Comment:
This is indexed with one because I did not put thought into making the FS algorithm run properly for multiple ct scenarios. If the FS code is supposed to be compatible with multiple scenarios, I think I have to do some more changes, starting with changing the 1 into an i. However, before I do this, is it generally interesting to implement the multiple scenario funcitionality into the FS algorithm?

The reason for using all radiological depths is that the coordinate projection and the associated interpolation, at the moment, uses the complete radiological depth cube.